Ports/ssmtp/patches/0002-We-dont-have-u_int32_t-but-do-have-uint32_t-so-repla.patch

```diff
@@ -0,0 +1,138 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fabian Dellwing <fabian.dellwing@gmail.com>
+Date: Tue, 25 Apr 2023 15:37:42 +0200
+Subject: [PATCH] We dont have `u_int32_t` but do have `uint32_t`, so replace
+ all to build successfull.
+
+---
+ md5auth/md5.h  |  4 ++--
+ md5auth/md5c.c | 36 ++++++++++++++++++------------------
+ 2 files changed, 20 insertions(+), 20 deletions(-)
+
+diff --git a/md5auth/md5.h b/md5auth/md5.h
+index 0f4d73e6d151c08dfd538b3e50b86cf4b0fd4c5d..9bbd41d4f2716821204f7e1190484f76ba286c16 100644
+--- a/md5auth/md5.h
++++ b/md5auth/md5.h
+@@ -25,8 +25,8 @@ documentation and/or software.
+ 
+ /* MD5 context. */
+ typedef struct {
+-  u_int32_t state[4];                                   /* state (ABCD) */
+-  u_int32_t count[2];        /* number of bits, modulo 2^64 (lsb first) */
++  uint32_t state[4];                                   /* state (ABCD) */
++  uint32_t count[2];        /* number of bits, modulo 2^64 (lsb first) */
```

Comment:
Instead of replacing all these types, can't you just do a single `typedef` in the header?